core-feature-pack/src/main/resources/content/bin/domain.bat

```diff
@@ -99,6 +99,29 @@ if "x%JAVA_HOME%" == "x" (
   )
 )
 
+"%JAVA%" --add-modules=java.se -version >nul 2>&1 && set MODULAR_JDK=true || set MODULAR_JDK=false
```

Comment:
In order to have it work in jconsole.bat, I had to add parenthesis when setting MODULAR_JDK var.
&& (set MODULAR_JDK=true) || (set MODULAR_JDK=false)